iot_relay_server/app/main.py

```
@@ -191,22 +191,45 @@ async def override_control(data: OverrideIn) -> dict:
+        return {
+            "decisions": decisions,
+            "agent_enabled": ai_agent.agent_enabled,
+            "input": sensors_dict,
+            "debug": {
+                "has_api_key": bool(settings.OPENROUTER_API_KEY),
+                "model": settings.AI_AGENT_MODEL,
+                "enabled_phases": ai_agent.ENABLED_PHASES,
+                "last_llm_call": str(ai_agent._last_llm_call) if ai_agent._last_llm_call else None,
+            },
+        }
     except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)